docs/eventing/debugging/example.yaml

```diff
@@ -105,6 +108,44 @@ spec:
           ports:
             - containerPort: 8080
 
+
+---
+
+# Second Subscription to the Channel for event-display
```

Comment:
Rather than make a second Subscription and Service, replace the existing one with event-display. Additionally, I like the use of K8s Service + Deployment here, rather than Knative Service as it makes it easier to describe what is happening and removes a place for potential debugging. It also makes the logs last 'longer', but not having the Pods automatically get scaled away.

Overall, I recommend simply changing the image on the existing Deployment, rather than add these new objects.